lib/vanagon/platform/windows.rb

```diff
@@ -1,6 +1,7 @@
 class Vanagon
   class Platform
     class Windows < Vanagon::Platform
+      attr_accessor :proj_root
```

Comment:
It's OK to assume that the `project` object is available; all Vanagon `component` objects exist only in relation to a `project` object.